site/source/conf.py

```diff
@@ -83,7 +83,7 @@ def skip_member(app, what, name, obj, skip, options):
         return name.startswith('_')
 
 def replace(app, what, name, obj, options, lines):
-    for i in range(len(lines)):
+    for i, value in enumerate(lines):
         if not "'|n'" in lines[i]:
             if not "'|s'" in lines[i]:
                 lines[i] = lines[i].replace("|n", "\n").replace("|s", "  ")
```

Comment:
```suggestion
    for i, value in enumerate(lines):
        lines[i] = lines[i].replace("|n", "\n").replace("|s", "  ")
```
- There is no point in checking twice and then running 2 more times across the string
- If these symbols are absent, there is no point in replacing them

Reply:
I did this so that `'|n'` and `'|s'` would not be replaced, for example in this case https://tosmanov.github.io/datumaro/api/api/datumaro.cli.html#id3
If do as you suggest, the result will be as follows:
![image](https://user-images.githubusercontent.com/54434686/141059112-5dc3af22-0923-4c92-a4a1-4aadd0e95e58.png)

Reply:
Okay, I missed that moment.